packages/components/src/dropdown-menu/README.md

```diff
@@ -1,8 +1,60 @@
-# Dropdown Menu
+# DropdownMenu
 
-Dropdown Menu is a React component to render an expandable menu of buttons. It is similar in purpose to a `<select>` element, with the distinction that it does not maintain a value. Instead, each option behaves as an action button.
+The DropdownMenu displays a list of actions (each contained in a MenuItem, MenuItemsChoice, or MenuGroup) in a compact way. It appears in a Popover after the user has interacted with an element (a button or icon) or when they perform a specific action. 
 
-## Usage
+![An expanded DropdownMenu, containing a list of MenuItems.](https://wordpress.org/gutenberg/files/2019/01/DropdownMenuExample.png)
+
+## Table of contents
+
+1. [Design guidelines](#design-guidelines)
+2. [Development guidelines](#development-guidelines)
+
+## Anatomy
+
+![Anatomy of a DropdownMenu.](https://wordpress.org/gutenberg/files/2019/01/DropdownMenuAnatomy.png)
+
+1. Popover: a container component in which the DropdownMenu is wrapped.
+2. Parent button: the icon or button that is used to toggle the display of the Popover containing the DropdownMenu.
+3. MenuItem: the list items within the DropdownMenu.
+
+## Design guidelines
+
+### Usage
+
+#### When to use a DropdownMenu
+
+Use a DropdownMenu when you want users to:
+
+- Choose an action or change a setting from a list, AND
+- Only see the available choices contextually.
+
+If you need to display all the available options at all times, consider using a Toolbar instead.
+
+![Use a DropdownMenu to display a list of actions after the user interacts with an icon.](https://wordpress.org/gutenberg/files/2019/01/DropdownMenuDo.png)
+
+**Do**
+Use a DropdownMenu to display a list of actions after the user interacts with an icon.
+
+![Don’t use a DropdownMenu for important actions that should always be visible. Use a Toolbar instead.](https://wordpress.org/gutenberg/files/2019/01/DropdownMenuDont.png)
+
+**Don’t**
+Don’t use a DropdownMenu for frequently used actions. Use a Toolbar instead.
+
+#### Behavior
+
+The parent button should (have|not have) a triangular icon to the right of the icon or text to indicate that interacting with it will show a DropdownMenu. 
```

Comment:
I think this `have|not have` needs to be clarified. Some of our examples use the dropdown icon, while others (like the ellipsis menu on the block toolbar) do not. I think the guideline we're looking for is something along the lines of: 

> Generally, the parent button should have a triangular icon to the right of the icon or text to indicate that interacting with it will show a DropdownMenu. In rare cases where the parent button directly indicates that there'll be more content (through the use of an ellipsis or "More" label), this can be omitted.

Reply:
Good call 👍